<?xml version="1.0"?>
<launch>
    <rosparam file="$(find test_hardware_interface)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find test_hardware_interface)/config/controllers.yaml" command="load"/>
    <rosparam file="$(find test_hardware_interface)/config/joint_limits.yaml" command="load"/>
    <node name="test_hardware_interface" pkg="test_hardware_interface" type="test_hardware_interface_node" output="screen"/>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
	args="
		/test/controller/state
		/test/controller/position/robot_joint_1
		/test/controller/position/robot_joint_2
	"/>
</launch>
